requirements.txt

```diff
@@ -1,4 +1,4 @@
-ansible==2.8.8
+ansible==2.9.1
```

Comment:
Can you downgrade and try again? It might be related?

Reply:
Reverted here: https://github.com/elastic/beats-tester/pull/208/commits/6a21cb009ef87f9d6fa5a31c025423d4b5af9837

Reply:
I kept the version as it was impossible to run the test suite on Mac with 2.8.8

I checked that the very same error than with 2.8.8 is happening with 2.9.1, see https://github.com/elastic/beats-tester/pull/208/files#r761364594 to understand why